cfgov/search/views.py

```diff
@@ -54,6 +56,14 @@ def post(self, request):
         })
 
 
+def remove_duplicates(pages):
+        seen = set()
```

Comment:
Can this please be made indented by 4 spaces to match consistency with the rest of the file?